CodeEdit/Features/WindowCommands/ViewCommands.swift

```diff
@@ -86,32 +72,13 @@ struct ViewCommands: Commands {
 
             Divider()
 
-            Button("\(navigatorCollapsed ? "Show" : "Hide") Navigator") {
```

Comment:
Instead of a default value here (and the changes to the initializer in the window controller), would an `if let ` check work instead?
Eg:
```swift
if let windowController {
    HideCommands(...)
} else {
    // disabled buttons
}
```

Reply:
Hm...I can't think of a good way to go about this without some pretty nasty code duplication though; is there a way to do this without say both a `HideCommands` and a redundant `DisabledHideCommands` class?

Reply:
That's a good point, I was nervous about having an extra initializer for WindowController that would put it into an invalid state. But I also realize now that the properties on the controller are already optional so I'm less worried about it.